draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -260,18 +247,13 @@ changes to the application itself.
 
 ## Terminology {#terminology}
 
-Throughout this document we make use of terms that are either specific
+This document uses terms that are either specific
 for multipath transport or are defined in the context of MP-DCCP,
 similar to {{RFC8684}}, as follows:
  
 Path: A sequence of links between a sender and a receiver, defined in
 this context by a 4-tuple of source and destination address/ port pairs.
 
-Subflow: A flow of DCCP segments operating over an individual path,
-which forms part of a larger MP-DCCP connection. A subflow is started
-and terminated similar to a regular (single-path) DCCP connection. The term subflow
-can also be used to refer to an MP-DCCP connection with a single path.
-
```

Comment:
@gorryfair Can you please confirm that this deletion is ok after you added another paragraph for "subflow" below?

Reply:
Gorry agreed by email